cmd/snap/cmd_get.go

```diff
@@ -287,9 +287,9 @@ func (x *cmdGet) Execute(args []string) error {
 }
 
 func validateConfdbFeatureFlag() error {
-	if !features.Confdbs.IsEnabled() {
-		_, confName := features.Confdbs.ConfigOption()
-		return fmt.Errorf(`the "confdbs" feature is disabled: set '%s' to true`, confName)
+	if !features.Confdb.IsEnabled() {
+		_, confName := features.Confdb.ConfigOption()
+		return fmt.Errorf(`the "confdb" feature is disabled: set '%s' to true`, confName)
```

Comment:
While we are at it, this could use the `experimental.` prefix for clarity
```suggestion
		return fmt.Errorf(`the "experimental.confdb" feature is disabled: set '%s' to true`, confName)
```

Reply:
It's included in the next bit, so it looks like `the "confdb" feature is disabled: set 'experimental.confdb' to true`